wasm/src/main/java/com/dylibso/chicory/wasm/types/MemoryLimits.java

```diff
@@ -44,14 +81,27 @@ public static MemoryLimits defaultLimits() {
         return DEFAULT_LIMITS;
     }
 
+    /**
+     * {@return the initial size, in pages}
+     */
     public int initialPages() {
         return initial;
     }
 
+    /**
+     * {@return the maximum size, in pages}
+     */
     public int maximumPages() {
         return maximum;
     }
 
+    /**
+     * {@return <code>true</code> if the limits apply to a shared memory segment, or <code>false</code> otherwise}
+     */
+    public boolean shared() {
```

Comment:
I'm failing to find a reference to this field in the spec:
https://webassembly.github.io/spec/core/search.html?q=shared
Do you have a link?

Reply:
Sure. https://github.com/WebAssembly/threads/blob/main/proposals/threads/Overview.md#spec-changes

Reply:
Oh, this is coming from the threads proposal, I'm fine with it, but please add this link in a comment, otherwise LGTM.

Reply:
Done.